### PR TITLE
gldas ts make 

### PR DESCRIPTION
**Pull request by jread-usgs**
need to hop over to https://github.com/USGS-R/mda.streams/issues/269, but hoping you could 👍 or :-1: this @aappling-usgs

### CODE REVIEW

File: 1_timeseries/code/stage_ts.R [hunk 1]
```diff
@@ -33,6 +33,23 @@ stage_ts <- function(ts.file){
       ts.table$no.data[use.i] <- !files %in% processed.files
     } #// else do nothing
     
+  } else if (src == 'nldas') {
```
Comment (aappling-usgs) [inline 1_timeseries/code/stage_ts.R]: did you mean `} else if (src == 'gldas') {` ?

File: lib/auth_from_profile.R [hunk 2]
```diff
@@ -4,5 +4,5 @@
 #'   located
 auth_from_profile = function(filename=file.path(Sys.getenv("HOME"), ".R", "stream_metab.yaml")){
   profile <- load_profile(filename)
-  authenticate_sb(profiles$sb_user, profiles$sb_password)
+  authenticate_sb(profile$sb_user, profile$sb_password)
```
Comment (aappling-usgs) [inline lib/auth_from_profile.R]: oops, thanks

**Comment by aappling-usgs** [comment]
looks good except probable typo noted in first comment

File: 1_timeseries/code/stage_ts.R (hunk 1 above)
Comment (jread-usgs) [inline 1_timeseries/code/stage_ts.R]: 👍 
I'll grab that. Let's leave this open until I get it.

Comment (aappling-usgs) [inline 1_timeseries/code/stage_ts.R]: ok

**Comment by jread-usgs** [comment]
should be good now

**Pull request merged**